src/data/general-cs-quiz.ts

```diff
@@ -1414,33 +1413,55 @@ const generalCSQuiz = [
-    Answer: "O(n)" ,
-    Distractor1: "O(n^2)" ,
-    Distractor2: "O(1)" ,
-    Distractor3: "O(log n)" ,
-    Explanation: "The letter n represents the input size, and the big O represents the increase of the running time." ,
+    Question:
+      "What's the Big O notation of an algorithm that increases the running time linearly with the input size?",
+    Answer: "O(n)",
+    Distractor1: "O(n^2)",
+    Distractor2: "O(1)",
+    Distractor3: "O(log n)",
+    Explanation:
+      "The letter n represents the input size, and the big O represents the increase of the running time.",
     Link: "https://www.freecodecamp.org/news/big-o-cheat-sheet-time-complexity-chart/"
   },
   {
-    Question: "Why are sorting algorithms important in programming?" ,
-    Answer: "Sorting algorithms are important because they can reduce the complexity of a problem." ,
-    Distractor1: "Sorting algorithms are important because they are only used to arrange data in alphabetical order" ,
-    Distractor2: "Sorting algorithms are important because they are frequently asked in interviews." ,
-    Distractor3: "Sorting algorithms are not important because there are faster ways to solve problems" ,
-    Explanation: "Sorting algorithms are important because they can reduce the complexity of a problem by arranging data in a specific order, which can make it easier to solve and streamline various tasks and processes." ,
+    Question: "Why are sorting algorithms important in programming?",
+    Answer:
+      "Sorting algorithms are important because they can reduce the complexity of a problem.",
+    Distractor1:
+      "Sorting algorithms are important because they are only used to arrange data in alphabetical order",
+    Distractor2:
+      "Sorting algorithms are important because they are frequently asked in interviews.",
+    Distractor3:
+      "Sorting algorithms are not important because there are faster ways to solve problems",
+    Explanation:
+      "Sorting algorithms are important because they can reduce the complexity of a problem by arranging data in a specific order, which can make it easier to solve and streamline various tasks and processes.",
     Link: "https://www.freecodecamp.org/news/sorting-algorithms-explained-with-examples-in-python-java-and-c/#classification-of-a-sorting-algorithm"
-   }
-  
+  },
+  {
+    Question: "What is the kebab case format?",
+    Answer:
+      "Every word is lower case in grouped together words and separated by hyphens.",
+    Distractor1:
+      "Every word has the first letter capitalized in grouped together words.",
+    Distractor2:
+      "Every word is lower case in grouped together words and separated by underscores.",
+    Distractor3:
+      "Every word has the first letter capitalized except the first in grouped together words.",
+    Explanation:
+      "Kebab case has all words are lowercase, and each word gets separated by a dash.",
```

Comment:
I think this questions would work better to display an example

```suggestion
    Question: "Which of the following examples shows the correct syntax for kebab case?",
    Answer: "number-of-donuts",
    Distractor1: "number_of_donuts",
    Distractor2: "numberOfDonuts",
    Distractor3: "number*of*donuts",
    Explanation: "Kebab case separates each lowercase word by dashes.",
```

Reply:
Thank you for your suggestion. It definitely is a lot better to use an example in this case.